neuro-cli/tests/e2e/test_e2e_share.py

```diff
@@ -231,5 +231,5 @@ def test_add_grant_remove_role(request: Any, helper: Helper) -> None:
     captured = helper.run_cli(["acl", "list", "--full-uri", "-u", "public"])
     assert captured.err == ""
     result = [line.split() for line in captured.out.splitlines()]
-    assert [f"role://{role_name}", "read"] in result
+    assert [f"role://{role_name}", "read"] not in result
```

Comment:
Did it ever work before?

Reply:
Server behaviour changed recently. Now, if the role is removed, then we also remove permission to such a role.

Reply:
Understood